### PR TITLE
inigo_py: split extract middlewares to subpackages

### PR DESCRIPTION
**Pull request by kreyyser**
commit-id:d922910e

### CODE REVIEW

File: inigo_py/query.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        if output_len.value:
+            output_dict = json.loads(output_ptr.value[:output_len.value].decode("utf-8"))
+
+        ffi.disposeMemory(ctypes.cast(output_ptr, ctypes.c_void_p))
+        ffi.disposeHandle(self.handle)
+
+        return output_dict
+
+
+class InigoContext:
```
Comment (kolia-kaploniuk) [inline inigo_py/query.py:67]: looks like this one is not used anywhere anymore, let's clean this up

**Review by kolia-kaploniuk — APPROVED** [review]

File: example-flask/app.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from flask import Flask
 from flask_cors import CORS
 
-from inigo_py import FlaskMiddleware
+from inigo_py.flask import Middleware
 from middleware import LogBlockedInigoRequestsMiddleware, LogMiddleware, AuthMiddleware
```
Comment (kolia-kaploniuk) [inline example-flask/app.py:8]: basically this middleware doesn't work because InigoContext is not used

Comment (kolia-kaploniuk) [reply example-flask/app.py:8]: `LogBlockedInigoRequestsMiddleware`

Comment (kreyyser) [reply example-flask/app.py]: Auth middleware without InigoContext used also doesn't make sense. Removed those.

**Pull request merged**